app/components/List.jsx

```diff
@@ -4,8 +4,19 @@ import React from 'react';
 import ListItem from './ListItem';
 import ReactCSSTransitionGroup from './TimeoutTransitionGroup';
 import { mergeAndPrefix } from '../utils/stylePropable';
+import { List } from 'material-ui';
+
+export default class extends React.Component {
+  static propTypes = {
```

Comment:
Yep but the problem with this solution is that the component name e.g. `List` is the same as the one from `material-ui`, so there will be a conflict :cry: 

That's why I use static in this component and not on the other one